dashboard/client/pages/index.tsx

```diff
@@ -12,20 +12,32 @@ interface IProps {
   tickets: any[]
 }
 
-const HomePage: NextStatelessComponent<IProps> = ({ spaces, tickets }) => (
-  <Layout spaces={spaces}>
-    {tickets.map(t => (
-      <Ticket key={t.title} ticket={t} />
-    ))}
-  </Layout>
-)
+const HomePage: NextStatelessComponent<IProps> = ({ spaces, tickets: initalTickets, query }) => {
+  const [tickets, setTickets] = useState(initalTickets)
+
+  useEffect(() => {
+    const fetchTickets = async () => {
```

Comment:
do we rly need this outer function `fetchTickets`?

Reply:
you cant put an async directly on useEffect bc useEffect should either return a cleanup function or nothin according to the docs, it'll give u a lil warning

Reply:
good to know!